iris/tests/pipeline/chat/ask_user_pipeline/test_ask_user_multiple_questions.py

```
@@ -0,0 +1,136 @@
+        )
+        cls.keywords_task = extract_keywords(cls.template_concatenated, cls.task)
+
+        pipeline = AskUserPipeline()
+
+        cls.dto = copy.deepcopy(DTO)
+        cls.message_time = FIRST_MESSAGE_TIME
+
+        cls.questions_all_tests = []
+        messages = []
+
+        for i in range(number_of_tests):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)